velox/functions/sparksql/DecimalUtil.h

```diff
@@ -211,5 +226,16 @@ class DecimalUtil {
     int32_t numOccupied = sizeof(A) * 8 - bits::countLeadingZeros(valueAbs);
     return numOccupied + kMaxBitsRequiredIncreaseAfterScaling[aRescale];
   }
+
+  /// If we have a number with 'numLeadingZeros' leading zeros, and we scale it
```

Comment:
I'm not sure I understand what this means. Other readers may be confused as well. Would you clarify?

Reply:
Revised the comments.